db/init/init.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE IF NOT EXISTS save_cloud;
```

Comment:
with `createDatabaseIfNotExist` -- looks like we don't need it

Reply:
This script is not used by liquibase but rather run directly by MySQL instance in a container, so I think there is no problem here

Reply:
I agree. It's not needed